lib/sycamore/sycamore/transforms/text_extraction/ocr_models.py

```diff
@@ -109,9 +111,9 @@ def __init__(self):
 
         self.pytesseract = pytesseract
 
-    def get_text(self, image: Image.Image) -> str:
+    def get_text(self, image: Image.Image) -> tuple[str, float]:
         val = self.pytesseract.image_to_string(image)
-        return val
+        return val, 0.0
```

Comment:
Can you add a comment that is not implemented? If it's not possible to calculate it from the output of tesseract, I would rather we return `None` to indicate this is not a valid font size

Reply:
done